collections/ansible_collections/demo/openshift/roles/eda_controller/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+galaxy_info:
+  role_name: eda_controller
+  author: Mitesh Sharma (mitsharm@redhat.com)
```

Comment:
is this role being copied from somewhere else?  if so, is it possible to just add the role/collection to the product demos EE instead of adding it to this repo?  otherwise we have to maintain the copy over time.

Reply:
It's from here, https://github.com/redhat-cop/agnosticd/tree/development/ansible/roles_ocp_workloads/ocp4_workload_eda_controller

Reply:
ok since that isn't much of a moving target we can just keep our own "fork" of the role for now